### ISSUE TITLE
c2ta changed?

### ISSUE THREAD

**Issue by dlomax64** [ssc-oscar/DRE#81]
@audrism Seems that the c2ta mapping does not work anymore. Has the syntax for it changed?

PS. I fixed the issue you emailed me about and will push tonight, just need to know how to fix the c2ta problem first

**Comment by dlomax64** [comment]
Okay, I will update it this week

**Comment by audrism** [comment]
I do not see any c2ta or c2dat in the mappings, have they been removed?

c2ta is just an attribute of a commit, not really a map, however.

**Comment by dlomax64** [comment]
> c2ta is now c2dat, so it has more fields:
> 
> > echo ffc35247f350f09edf144241cc13f96107cde25c | ~/lookup/getValues c2dat
> > ffc35247f350f09edf144241cc13f96107cde25c;1516957415;+0700;Nikolay Ryabkov [nikolay.ryabkov@sibers.com](mailto:nikolay.ryabkov@sibers.com);9e7aeb82e8a5151a023fde04cf481726e89e0c43;cab47b0b2c915e6e8594c8fcb0849eb130bdc577

What are the indexes of this? is it `commit;time;author;tree;parent`?

**Comment by audrism** [comment]
correct:  but timezone is after time also parents (if several) are separated by :

**Comment by dlomax64** [comment]
c2dat doesn't work in my docker container. Any idea why? It works when I'm on any of the servers outside of the container.

![image](https://user-images.githubusercontent.com/53716031/155229987-01a2e5ae-c4c5-42ef-aebe-4dff54ea7f67.png)

**Comment by audrism** [comment]
Nothing should work inside container: it does ssh da5 from the container to run the command

**Comment by dlomax64** [comment]
Right, I forgot about that. For some reason the ssh wasn't working either, but it looks like it is now
